src/components/SavedPage/SavedPage.js

```diff
@@ -0,0 +1,41 @@
+import React, { useState, useEffect } from "react";
+import SavedCard from "../SavedCard/SavedCard";
+
+export const SavedPage = () => {
+  const [savedCards, setSavedCards] = useState([]);
+
+  const getFromLocalStorage = () => {
+    const keys = Object.keys(localStorage);
+    let cards;
+    if (keys.length) {
+      cards = keys.map((key) => {
+        return JSON.parse(localStorage.getItem(key));
+      });
+      setSavedCards([...cards]);
+    }
+  };
+
+  const removeFromStorage = (id) => {
+    localStorage.removeItem(id);
+    const remainingSaved = savedCards.filter((card) => id !== card.id);
+    setSavedCards([...remainingSaved]);
+    window.location.reload(false);
+  };
+
+  useEffect(() => {
+    getFromLocalStorage();
+  }, []);
+
+  const cards = savedCards.map((card) => (
+    <SavedCard card={card} remove={removeFromStorage} />
+  ));
+
+  return (
+    <>
```

Comment:
I would create some sort of semantic tag to wrap this return in, even if its just a div!